app/controllers/v3/apps_controller.rb

```diff
@@ -37,6 +37,7 @@ class AppsV3Controller < ApplicationController
   def index
     message = AppsListMessage.from_params(query_params)
     invalid_param!(message.errors.full_messages) unless message.valid?
+    # add_warning_headers(message.warnings) if message.warnings.size > 0
```

Comment:
Is this meant to be commented-out?

Reply:
I should delete it and make it a separate story. Currently there are no warnings, but what's an error in a set-metadatum should just be a warning in a query.

Reply:
On second thought, a query like `!<10 MB long name>` should continue to raise an error rather than a warning because executing a lot of such queries in parallel would create a DOS. Better to continue to error-out on any invalid name.